env/web_env.go

```diff
@@ -90,7 +90,9 @@ func getEnvValueFromHTTP(urlStr, envKey string) (string, string, string, error)
 		return "", "", "", err
 	}
 
-	ctx, cancel := context.WithTimeout(context.Background(), 3*time.Second)
+	// Adding a timeout of 6.5 seconds to deal with k3s slow dns resolution caused in turn by
+	// CoreDNS 6 second default timeout.
+	ctx, cancel := context.WithTimeout(context.Background(), 6.5*time.Second)
```

Comment:
should this be `6*time.Second`?

Reply:
If this is only called once - this is what it looks like - you can even set it to one minute.

Reply:
We don't want our CLI to blocked for such a long time @vadmeste this is specifically to fix a k3s DNS resolution bug.